lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -515,17 +522,28 @@ private JourneyResponse buildReuseResponse(
                         vcBundle.credentials, attainedVot.getProfileType()),
                 auditEventUser.getSessionId(),
                 false);
-        migrateCredentialsToEVCS(userId, vcBundle, evcsAccessToken);
+        migrateCredentialsToEVCS(
+                auditEventUser,
+                deviceInformation,
+                ipvSessionItem.getVot(),
+                vcBundle,
+                evcsAccessToken);
 
         return vcBundle.isPendingEvcsIdentity ? JOURNEY_REUSE_WITH_STORE : JOURNEY_REUSE;
     }
 
     private void migrateCredentialsToEVCS(
-            String userId, VerifiableCredentialBundle vcBundle, String evcsAccessToken)
-            throws EvcsServiceException, VerifiableCredentialException {
-        if (configService.enabled(EVCS_WRITE_ENABLED) && !vcBundle.isEvcsIdentity) {
+            AuditEventUser auditEventUser,
+            String deviceInformation,
+            Vot vot,
+            VerifiableCredentialBundle vcBundle,
+            String evcsAccessToken)
+            throws EvcsServiceException, VerifiableCredentialException, SqsException {
+        if (configService.enabled(EVCS_WRITE_ENABLED)) {
```

Comment:
You've lost a condition on `vcBundle.isEvcsIdentity` in the merge resolution here

Reply:
Also added a scenario when its evcsIdentity (with f2f vc and pending_return) then do f2f completion vcs updation pending_return -> current.